SimplyLinkedList.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+from Link import Link
```

Comment:
The `Link` is an implementation detail and can be kept into the same file as the where `SimplyLinkedList` is.